README.md

```diff
@@ -127,6 +127,10 @@ sudo apt install texlive texlive-latex-extra texlive-fonts-extra texlive-latex-r
 > Note: this installation may take up a lot of space. The developers are working on providing a simpler, lighter LaTeX package for you to install
 2. You can check you did it right by running `latex`
 
+#### `pip3` Installation
+1. Install `pip3` in the `python3-pip` package with your package manager: `sudo apt-get install python3-pip`
+2. You can check you did it right by running `pip3`
```

Comment:
nitpick: better imo
```suggestion
2. You can check if you did it right by running `pip3`
```

Reply:
I think we left out the `if` in a few other places as well. Should prolly fix that, though maybe not in this PR.

Reply:
the *you can check* language is correct without the **if** (idiomatically speaking). the conditional is implied by *check*, making the **if** redundant.

Reply:
Well, kinda. Without the `if`, it's more like "you can check that you did it right", which gives some sort of implication that it is expected that it went right. With `if` it can go both ways. It's not wrong, I just felt like this would be better yk

Reply:
I think both are sufficiently correct. I believe the second one, including the `if` is better. In technical documents such as software documentation, it never hurts to be redundant.